<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoldMachineClass</class>
 <widget class="QMainWindow" name="MoldMachineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注塑过程监控系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widgetAll" native="true">
      <layout class="QGridLayout" name="gridLayout_5" rowstretch="8,2">
       <item row="0" column="0">
        <widget class="QWidget" name="widgetDisImg" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelBeforeOne">
            <property name="styleSheet">
             <string notr="true">background-color:black</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelAfterOne">
            <property name="styleSheet">
             <string notr="true">background-color:black</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1,1">
          <item row="0" column="1">
           <widget class="QLabel" name="labelDisCurProcess">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="text">
             <string>系统未启动</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="progressBarStep">
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::TopToBottom</enum>
            </property>
            <property name="format">
             <string>%p%</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QWidget" name="widgetDisIO" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBoxIOState">
               <property name="title">
                <string>输入状态</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelInS1">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_red_20x20.jpg</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>false</bool>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="labelInS2">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_red_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="labelInS3">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_red_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="labelInS4">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_red_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4">
                 <widget class="QLabel" name="labelInS5">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_red_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="5">
                 <widget class="QLabel" name="labelInS6">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_red_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBoxOutState">
               <property name="title">
                <string>输出状态</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QLabel" name="labelOutS1">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_green_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="labelOutS2">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_green_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="labelOutS3">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_green_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="labelOutS4">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_green_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4">
                 <widget class="QLabel" name="labelOutS5">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_green_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="5">
                 <widget class="QLabel" name="labelOutS6">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>Resources/ico/light_green_20x20.jpg</pixmap>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QTableWidget" name="tableWidgetDisRes">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <property name="rowCount">
             <number>0</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>25</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>85</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>25</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>30</number>
            </attribute>
            <attribute name="verticalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>序号</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>时间</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>结果</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
            <column>
             <property name="text">
              <string>信息</string>
             </property>
             <property name="textAlignment">
              <set>AlignCenter</set>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSaveCfgAsXML"/>
    <addaction name="actionReadXMLAsSqlData"/>
   </widget>
   <widget class="QMenu" name="menuRobin">
    <property name="title">
     <string>循环</string>
    </property>
    <addaction name="actionCycleTimesSetting"/>
    <addaction name="actionCycleStepSetting"/>
    <addaction name="actionCounterSetting"/>
    <addaction name="actionDelaySetting"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>图像</string>
    </property>
    <addaction name="actionImageSetting"/>
   </widget>
   <widget class="QMenu" name="menuIO">
    <property name="title">
     <string>输入/输出</string>
    </property>
    <addaction name="actionENetIOSetting"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actionMoldObjects"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRobin"/>
   <addaction name="menuImage"/>
   <addaction name="menuIO"/>
   <addaction name="menuRun"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionENetIOSetting">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/IO.png</normaloff>Resources/ico/IO.png</iconset>
   </property>
   <property name="text">
    <string>输入/输出属性</string>
   </property>
  </action>
  <action name="actionImageSetting">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/pic.png</normaloff>Resources/ico/pic.png</iconset>
   </property>
   <property name="text">
    <string>图像设置</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/run.png</normaloff>Resources/ico/run.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="actionCycleTimesSetting">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/times.png</normaloff>Resources/ico/times.png</iconset>
   </property>
   <property name="text">
    <string>循环次数</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/about.png</normaloff>Resources/ico/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/stop.png</normaloff>Resources/ico/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionCycleStepSetting">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/step.png</normaloff>Resources/ico/step.png</iconset>
   </property>
   <property name="text">
    <string>循环步骤</string>
   </property>
  </action>
  <action name="actionCounterSetting">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/count.png</normaloff>Resources/ico/count.png</iconset>
   </property>
   <property name="text">
    <string>计数器</string>
   </property>
  </action>
  <action name="actionDelaySetting">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/delay.png</normaloff>Resources/ico/delay.png</iconset>
   </property>
   <property name="text">
    <string>延时器</string>
   </property>
  </action>
  <action name="actionSaveCfgAsXML">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/save.png</normaloff>Resources/ico/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionReadXMLAsSqlData">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/load.png</normaloff>Resources/ico/load.png</iconset>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="actionMoldObjects">
   <property name="icon">
    <iconset>
     <normaloff>Resources/ico/objects.png</normaloff>Resources/ico/objects.png</iconset>
   </property>
   <property name="text">
    <string>模具对象</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MoldMachine.qrc"/>
 </resources>
 <connections/>
</ui>
